File: strategy/php/mutequack.php
<?php
/**
 * file mutequack.php
 *
 * @author kujaomega
 */

/**
 * This class is for the ducks that don't quack.
 *
 * Class MuteQuack
 */
class MuteQuack implements QuackBehavior
{
    /**
     * This is the quack method
     */
    public function quacks()
    {
        echo( ' <<Silence>> ');
    }
}

File: strategy/java/MuteQuack.java
/**
 * file MuteQuack.java
 * 
 * @author kujaomega
 */

/**
 * This is the mutequack class that the quack is muted.
 * 
 * class MuteQuack
 */
public class MuteQuack implements QuackBehavior{
    public void quack(){
        System.out.println("<<Silence>>");
    }
}
